src/components/card/Card.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import "./cardStyle.css";
+import URLs from"../../utils/constants.js";
+const Card = () => {
+  const [countriesData, getData] = useState([]);
+  const fetchUserData = () => {
+    fetch(URLs.getAllCountries)
+      .then((response) => {
+        return response.json();
+      })
+      .then((data) => {
+        getData(data);
+      });
+  };
+  useEffect(() => {
+    fetchUserData();
+  }, []);
+
+  const navigate = useNavigate();
+  const navigateToContacts = () => {
+    navigate("/country-detail");
+  };
+  
+  const test = countriesData.map((country,index) => {
```

Comment:
the test is not a perfect naming convention. You can also directly map countries data in return of this component.